python_modules/dagster-ext/dagster_ext/__init__.py

```diff
@@ -114,6 +114,17 @@ class ExtDataProvenance(TypedDict):
 ]
 
 
+# ##### JSON SCHEMA
+
+_schema_root = os.path.join(os.path.dirname(__file__), "../json_schema")
+
+CONTEXT_JSON_SCHEMA_PATH = os.path.join(_schema_root, "context.json")
+with open(CONTEXT_JSON_SCHEMA_PATH) as f:
+    CONTEXT_JSON_SCHEMA = json.load(f)
+MESSAGE_JSON_SCHEMA_PATH = os.path.join(_schema_root, "message.json")
+with open(MESSAGE_JSON_SCHEMA_PATH) as f:
+    MESSAGE_JSON_SCHEMA = json.load(f)
```

Comment:
synchronously doing I/O on module import very non-ideal. Can we restructure to do on demand and cache?

Reply:
done